modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -6,7 +6,23 @@ endif::[]
 The Mule Maven plugin enables you to integrate the packaging, testing, and deployment of your Mule applications with your Maven lifecycle. +
```

Comment:
I don't think that this topic set is titled correctly, because a Reference is typically a set of one table or list after another, comprising isolated data but not user actions or conceptual information.
Is this really About the Mule Maven Plugin, or Mule Maven Plugin Overview, or What the Mule Maven Plugin Does, or Using the Mule Maven Plugin? 
What is the focus of the set of topics? What encompasses all of the subtopic titles?

Reply:
The topic set introduction leads me to believe that this topic is going to focus on 
Packaging 
Testing
Deploying

But these are not the subtopics that I see below.
Please consider retitling and reintroducing based on the information actually contained within.

Reply:
I see that this topic set is parallel with that of PR #1014.
Please apply my edit suggestions to that PR here as well; I will not try to reproduce them here.
The following is proofreading only.